draft-ietf-gnap-core-protocol.md

```diff
@@ -1899,54 +1688,85 @@ properties.
-              from what was requested by the client instance.
+    associated with this access token, as defined in 
+    {{resource-access-rights}}. If included, this MUST reflect the rights
+    associated with the issued access token. These rights MAY vary
+    from what was requested by the client instance.
 
 expires_in (integer)
 : OPTIONAL. The number of seconds in
-              which the access will expire. The client instance MUST NOT use the access
-              token past this time. An RS MUST NOT accept an access token
-              past this time. Note that the access token MAY be revoked by the
-              AS or RS at any point prior to its expiration.
-
-key (object / string / boolean)
-: REQUIRED. The key that the token is bound to. If the boolean value `true` is used,
-              the token is bound to the [key used by the client instance](#request-client) in its request 
-              for access. If the boolean value `false` is used,
-              the token is a bearer token with no key bound to it.
-              Otherwise, the key MUST be an object or string in a format
-              described in {{key-format}}, describing a public key to which the
-              client instance can use the associated private key. The client instance MUST be able to
-              dereference or process the key information in order to be able
-              to sign the request.
-
-The following non-normative example shows a single bearer token with a management
-URL that has access to three described resources.
+    which the access will expire. The client instance MUST NOT use the access
+    token past this time. An RS MUST NOT accept an access token
+    past this time. Note that the access token MAY be revoked by the
+    AS or RS at any point prior to its expiration.
+
+key (object / string)
+: OPTIONAL. The key that the token is bound to, if different from the
+    client instance's presented key. The key MUST be an object or string in a format
+    described in {{key-format}}, describing a public key to which the
+    client instance can use the associated private key. The client instance MUST be able to
+    dereference or process the key information in order to be able
+    to sign the request.
+
+durable (boolean)
+: OPTIONAL. Flag indicating a hint of AS behavior on token rotation.
+    If this flag is set to the value `true`, then the client instance can expect
+    a previously-issued access token to continue to work after it has been [rotated](#rotate-access-token)
+    or the underlying grant request has been [modified](#continue-modify), resulting
+    in the issuance of new access tokens. If this flag is set to the boolean value `false`
+    or is omitted, the client instance can anticipate a given access token
+    will stop working after token rotation or grant request modification.
+    Note that a token flagged as `durable` can still expire or be revoked through
+    any normal means.
+
+split (boolean)
```

Comment:
This is not very useful. If I request "L1", "L2" and "L3", and I get back L1, L4, L5, L6, L7 where everything but L1 is marked as "split", what do I make of the response? Do I try to correlate the 4 access tokens in the response with L2 and L3 in the request? How?

Reply:
The "split token" function is being tracked in other existing issues: #14 and #37, this PR just packages the existing function. Conversation on changing that function can happen on those issues and be addressed with a new PR.